sdk/communication/Azure.Communication.Common/src/CommunicationIdentifier.cs

```diff
@@ -27,6 +27,31 @@ public abstract class CommunicationIdentifier : IEquatable<CommunicationIdentifi
         public override bool Equals(object obj)
             => obj is CommunicationIdentifier other && Equals(other);
 
+        /// <summary>
+        /// Overrides the equality operator.
+        /// </summary>
+        /// <param name="left">The first identifier to compare.</param>
+        /// <param name="right">The second identifier to compare.</param>
+        /// <returns>True if the types and <see cref="RawId"/> match.</returns>
+        public static bool operator ==(CommunicationIdentifier left, CommunicationIdentifier right)
+        {
+            return Equals(left, right) && left.GetType() == right.GetType();
```

Comment:
nit: you can use an expression body here and elsewhere =>

Reply:
👍